logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultAnyAssertions.kt

```diff
@@ -52,4 +53,13 @@ class DefaultAnyAssertions : AnyAssertions {
         container.changeSubject.reportBuilder()
             .downCastTo(subType)
             .build()
+
+    override fun <T> isNotIn(container: AssertionContainer<T>, expected: List<T>): Assertion {
+        val assertions = expected.map { assertionBuilder.representationOnly.failing.withRepresentation(it).build() }
```

Comment:
```suggestion
        val assertions = expected.map { 
          assertionBuilder.representationOnly
            .withTest { /* TODO your assertion logic hear instead of hard coded */ false }
            .withRepresentation(it)
            .build() 
        }
```